network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/featured_blogs.html

```diff
@@ -0,0 +1,64 @@
+{% load wagtailimages_tags %}
+
+{% if page.featured_blogs.all.count > 0 %}
+{% with page.featured_blogs.all|dictsortreversed:"blog.first_published_at" as featured_blogs %}
+<div class="row">
+  <div class="col mb-4">
+    <hr class="mb-4 hr-emphasis mt-0">
+    <h4 class="h4-heading">Latest news</h4>
+  </div>
+</div>
+<div class="row mb-3">
+  {% for blog in featured_blogs %}
+  {% if forloop.counter is 1 %}
+  <div class="col-md-6 mb-0 pb-4">
+    <div>
+      {% image blog.blog.specific.get_meta_image width-540 %}
+    </div>
+    <div class="row">
+      <div class="col-12 full-bleed-xs">
+        <div class="key-item mx-2 mx-md-4 p-3">
+          <p class="h6-heading mb-2">
+            {% for category in blog.blog.specific.category.all %}
+            <a href="/blog/category/{{category.slug}}">{{ category }}</a>{% if not forloop.last %}, {% endif %}
+            {% endfor %}
+          </p>
+          <h5 class="mb-2 h3-heading">
+            <a href="{{article.news.link}}">{{blog.blog.title}}</a>
+          </h5>
+          <p class="body-small mb-2">{% if blog.blog.author %}by {{blog.blog.author}} on {% endif %} {% if blog.blog.first_published_at %}{{ blog.blog.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
+          <p>{{blog.blog.search_description}}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  {% endif %}
+  {% endfor %}
+
+  <div class="col-md-6 mb-0 pb-4">
```

Comment:
Should this col only be built if there's more than 1 featured blog item? E.g.
```
{% if featured_blogs|length > 1 %}
...
{% endif %}
```

Reply:
Yes but I don't know how we can do this in the template. 😬 

We want the 2nd, 3rd, and 4th blog to all be wrapped in one "col" div. e.g.,

```html
<div class="col-md-6 mb-0 pb-4">
  2nd blog
  3rd blog
  4th blog
</div>
```

So if we wrap the "col" div in the following condition block
```
{% if featured_blogs|length > 1 %}
...
{% endif %}
```

then we will get the following HTML back, which isn't what we want

```html
<div class="col-md-6 mb-0 pb-4">
  2nd blog
</div>
<div class="col-md-6 mb-0 pb-4">
  3rd blog
</div>
<div class="col-md-6 mb-0 pb-4">
  4th blog
</div>
```

Reply:
We can put the `{% if featured_blogs|length > 1 %}` outside of the div, and keep the code you already have, so:
```
  {% if featured_blogs|length > 1 %}
  <div class="col-md-6 mb-0 pb-4">
    {% for blog in featured_blogs %}
    {% if forloop.counter > 1 %}
    <div>
      <p class="h6-heading mb-2">
        {% for category in blog.blog.specific.category.all %}
        <a href="/blog/category/{{category.slug}}">{{ category }}</a>{% if not forloop.last %}, {% endif %}
        {% endfor %}
      </p>

     ...
  </div>
  {% endif %}
```